src/test/java/com/marizueva/laboratory/hw3/fluentpageobject/pages/CommonPageElements.java

```diff
@@ -0,0 +1,195 @@
+            + "/descendant::input[@id='name']")
+    protected WebElement loginField;
+
+    @FindBy(xpath = "//ul/li[@class='dropdown uui-profile-menu open']"
+            + "/descendant::input[@id='password']")
+    protected WebElement passwordField;
+
+    @FindBy(xpath = "//ul/li[@class='dropdown uui-profile-menu open']"
+            + "/descendant::span[text()='Enter']")
+    protected WebElement enterCredentials;
+
+    @FindBy(xpath = "//ul/li[@class='dropdown uui-profile-menu open']"
+            + "/descendant::span[@id='user-name']")
+    WebElement loggedUserName;
+
+    @FindBy(xpath = "//header//ul/li/a[@href='index.html'][1]")
+    WebElement headerItemHome;
+
+    @FindBy(xpath = "//header//ul/li[2]/a[1]")
+    WebElement headerItemContact;
+
+    @FindBy(xpath = "//header//ul/li/a[@class='dropdown-toggle'][1]")
+    WebElement headerItemService;
+
+    @FindBy(xpath = "//header//ul/li/a[@href='metals-colors.html'][1]")
+    WebElement headerItemMetalAndColor;
+
+    @FindBy(xpath = "//ul[@class='sidebar-menu left']")
+    WebElement leftSection;
+
+    @FindBy(xpath = "/html/body/footer")
+    WebElement pageFooter;
+
+
+    @FindBy(xpath = "//header//ul/li/a[@class='dropdown-toggle']"
+            + "/following-sibling::ul[@class='dropdown-menu']/li/"
+            + "a[text()='Support']")
+    WebElement subMenuFromHeaderSupportItem;
+
+    @FindBy(xpath = "//header//ul/li/a[@class='dropdown-toggle']"
+            + "/following-sibling::ul[@class='dropdown-menu']/li/a[text()='Dates']")
+    WebElement subMenuFromHeaderDatesItem;
+
+    @FindBy(xpath = "//header//ul/li/a[@class='dropdown-toggle']"
+            + "/following-sibling::ul[@class='dropdown-menu']/li/a[text()='Search']")
+    WebElement subMenuFromHeaderSearchItem;
+
+    @FindBy(xpath = "//header//ul/li/a[@class='dropdown-toggle']"
+            + "/following-sibling::ul[@class='dropdown-menu']/li"
+            + "/a[@href='complex-table.html']")
+    WebElement subMenuFromHeaderComplexTableItem;
+
+    @FindBy(xpath = "//header//ul/li/a[@class='dropdown-toggle']"
+            + "/following-sibling::ul[@class='dropdown-menu']/li"
+            + "/a[@href='simple-table.html']")
+    WebElement subMenuFromHeaderSimpleTableItem;
+
+    @FindBy(xpath = "//header//ul/li/a[@class='dropdown-toggle']"
+            + "/following-sibling::ul[@class='dropdown-menu']/li"
+            + "/a[@href='user-table.html']")
+    WebElement subMenuFromHeaderUserTableItem;
+
+    @FindBy(xpath = "//header//ul/li/a[@class='dropdown-toggle']"
+            + "/following-sibling::ul[@class='dropdown-menu']/li"
+            + "/a[@href='table-pages.html']")
+    WebElement subMenuFromHeaderTableWithPagesItem;
+
+    @FindBy(xpath = "//header//ul/li/a[@class='dropdown-toggle']"
+            + "/following-sibling::ul[@class='dropdown-menu']"
+            + "/li/a[@href='different-elements.html']")
+    WebElement subMenuFromHeaderDifferentElementsItem;
```

Comment:
Looks like that this elements are extra here.

Reply:
removed